### ISSUE TITLE
Add a new field to representations, `version`

### ISSUE THREAD

**Issue by ahankinson** [rism-digital/pae-code-spec#48]
The `version` field should be added to the various PAE representations, with values of `pe` or `pe2`.

If `version` is empty or missing, then `pe` is assumed.

What this looks like:

In multi-line strings:

```
@version:pe2
```

In JSON:

```json
{ "version": "pe2" }
```

In MARC21:

`$2pe2` (subject to approval and addition to the [Musical Incipit Scheme Codes](https://www.loc.gov/standards/sourcelist/musical-incipit.html) list.

For the single-string representation, we have a couple options:

 - Simply put `pe2` at the beginning; if the string starts with `pe2` then it's version 2; if it starts with `pe`, or starts with a clef, then it's version 1
 - Give it a delimiter at the beginning: `;pe2` or `;pe`. I don't think this is necessary, but it could help if the fields get out-of-order.

Related to #5
